sui/gateway.js

```diff
@@ -612,6 +670,20 @@ if (require.main === module) {
             testNewField(value, options);
         });
 
+    program
+        .command('pause')
+        .description('Pause the gateway')
+        .action((options) => {
+            mainProcessor(pause, [], options);
+        });
+
+    program
+        .command('unpause')
+        .description('Unpause the gateway')
+        .action((options) => {
+            mainProcessor(unpause, [], options);
+        });
+
```

Comment:
update `test-sui.yaml` to run pause/unpause for gateway and ITS. Want to ensure that this action always work, instead of finding issues during an emergency